### DISCUSSION TITLE
RE model training - Floating Point Exception (Core Dumped)

### DISCUSSION THREAD

**Discussion by ghost** [PaddlePaddle/PaddleOCR#10167] [Q&A]
Hello everyone, we are trying to experiment with PaddleOCR. We are trying KIE training in general. We are able to run SER training however, when we attempt to run RE configuration (given in the link below) we get `Floating Point exception (core dumped)` error. Error happens instantly without any iterations on data. Any help would be appreciated a lot. Thanks in advance.

https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.6/configs/kie/layoutlm_series/re_layoutxlm_xfund_zh.yml

Hardware configuration:
Nvidia Tesla T4 GPU and CUDA 11.7 on Ubuntu 20.04

**Comment by ynjang** [comment]
I resolved a problem by using lowercase to label
